tox.ini

```diff
@@ -4,23 +4,25 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py37,py38,py39,py310
+envlist =
+    py38
+    py39
+    python3.10
```

Comment:
How come this one needs to be spelled out?

Reply:
So I ran into this on my machine (macOS) using pyenv.  

`py310: skipped because could not find python interpreter with spec(s): py310`

I'll see if I can find the relative post.

Reply:
Interesting... that explains why i needed to manually create a softlink for `py310` on my machine!